scripts/split-e2e-tests.ts

```diff
@@ -154,10 +168,12 @@ const createJob = (os: OSType, jobIdx: number, runSolo = false): CandidateJob =>
   runSolo,
 });
 
-const getTestNameFromPath = (testSuitePath: string): string => {
+const getTestNameFromPath = (testSuitePath: string, region?: string): string => {
   const startIndex = testSuitePath.lastIndexOf('/') + 1;
   const endIndex = testSuitePath.lastIndexOf('.test');
-  return testSuitePath.substring(startIndex, endIndex).split('.e2e').join('').split('.').join('-');
+  const regionSuffix =
+    RUN_IN_ALL_REGIONS.find((allRegions) => testSuitePath === allRegions || testSuitePath.match(allRegions)) && region ? region : '';
+  return testSuitePath.substring(startIndex, endIndex).split('.e2e').join('').split('.').join('-').concat(regionSuffix);
```

Comment:
nit: The delimiter `-` can be added before the regionSuffix for a better reading